.eleventy.js

```diff
@@ -22,6 +38,13 @@ module.exports = (eleventyConfig, pluginOptions) => {
   eleventyConfig.on('eleventy.config', (config) => {
     templateConfig = config;
   });
+  
+  if (pluginOptions.emptyOutputOnRebuild) {
+    eleventyConfig.on('eleventy.before', () => {
+      const options = mergeOptions(directoriesConfig, pluginOptions);
+      rmDir(options.outputDir);
+    });
+  }
```

Comment:
I think we can enable this behavior always. Also we can just use `rmSync`, the directy will be created anyways if necessary.

```suggestion
  eleventyConfig.on('eleventy.before', () => {
    const options = mergeOptions(directoriesConfig, pluginOptions);
    fs.rmSync(options.outputDir, { recursive: true, force: true });
  });
```